servers/physics_2d/godot_joints_2d.cpp

```diff
@@ -170,21 +173,76 @@ bool GodotPinJoint2D::pre_solve(real_t p_step) {
 	if (B && dynamic_B) {
 		B->apply_impulse(P, rB);
 	}
+	// from https://github.com/slembcke/Chipmunk2D/blob/master/src/cpRotaryLimitJoint.c#L25
```

Comment:
```suggestion
	// Taken from https://github.com/slembcke/Chipmunk2D/blob/master/src/cpRotaryLimitJoint.c#L25
```
Unsure if this is needed explicitly, but if so it should be a permalink or it won't be relevant in the future

Reply:
Removing comment.